testfiles/gedcom70/age-all.ged

```diff
@@ -0,0 +1,278 @@
+﻿0 HEAD
+1 GEDC
+2 VERS 7.0
+0 @SIMPLE@ INDI
+1 EVEN when child
+2 AGE < 8y
+3 PHRASE child
+1 EVEN when CHILD
+2 AGE < 8y
+3 PHRASE CHILD
+1 EVEN when Child
```

Comment:
This made sense as a 5.5.1-to-7.0 conversion, but doesn't make sense as a 7.0 sample file. Why have the exact same structure twice with different `EVEN` payloads? And so on for most of the rest of the file: each valid AGE payload is repeated at least twice, some many more times than that.

Reply:
Updated to address this comment, but in so doing, I filed https://github.com/FamilySearch/GEDCOM/issues/420.
Would like to discuss that before deciding whether more changes to age-all.ged are needed.